sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -469,7 +478,42 @@ public void analyzeSentimentWithLanguage() {
                         sentenceSentiment.getConfidenceScores().getNegative());
                 }
             });
-        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#string-string
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#String-String
+    }
+
+    /**
+     * Code snippet for {@link TextAnalyticsAsyncClient#analyzeSentiment(String, String, AnalyzeSentimentOptions)}
+     */
+    public void analyzeSentimentWithLanguageWithOpinionMining() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#String-String-AnalyzeSentimentOptions
+        textAnalyticsAsyncClient.analyzeSentiment("The hotel was dark and unclean.", "en",
+            new AnalyzeSentimentOptions().setIncludeOpinionMining(true))
+            .subscribe(documentSentiment -> {
+                List<MinedOpinion> positiveMinedOpinions = new ArrayList<>();
+                List<MinedOpinion> mixedMinedOpinions = new ArrayList<>();
+                List<MinedOpinion> negativeMinedOpinions = new ArrayList<>();
+                for (SentenceSentiment sentenceSentiment : documentSentiment.getSentences()) {
+                    sentenceSentiment.getMinedOpinions().forEach(minedOpinion -> {
+                        TextSentiment aspectTextSentiment = minedOpinion.getAspect().getSentiment();
+                        if (NEGATIVE.equals(aspectTextSentiment)) {
+                            negativeMinedOpinions.add(minedOpinion);
+                        } else if (POSITIVE.equals(aspectTextSentiment)) {
+                            positiveMinedOpinions.add(minedOpinion);
+                        } else if (MIXED.equals(aspectTextSentiment)) {
+                            mixedMinedOpinions.add(minedOpinion);
```

Comment:
Don't think its a good idea to be making these lists, can't we directly add the content from L508-l513 here?

Applicable for all examples below.

Reply:
updated